output.tf

```diff
@@ -1,4 +1,24 @@
-#output "example" {
-#  value       = var.example
-#  description = "This is just an example of an output"
-#}
+output "created" {
+  value       = var.create
+  description = "Was the OCI user created"
+}
+
+output "user_ocid" {
```

Comment:
any value in including the user name and group name?

Reply:
Doesn't hurt. Let me add them.